### ISSUE TITLE
EEM trying to open ProperTree before downloading

### ISSUE THREAD

**Issue by gashishozavr** [ephraimkreighbaum/EasyEFIMaker#1]
When I'm trying to download ProperTree, I'm receiving this error:

_ProperTree encountered an error:
python: can't open file 'C:\\Users\\quasa\\Downloads\\EasyEFIMaker-main\\tools\\propertree\\ProperTree-master\\ProperTree.py': [Errno 2] No such file or directory_

Can you make more detailed error log in console box?

**Comment by realedk** [comment]
Yes, this is an issue I'm working on right now, its attempting to open the file before downloading it.
To "fix" this temporarily, delete the **propertree** folder from within the **tools** folder than click it again.

The application is not usable yet, but hopefully will be in the near future.
